trailblazer/store/crud/update.py

```diff
@@ -7,12 +7,6 @@
 class UpdateHandler(BaseHandler_2):
```

Comment:
why is this called `BaseHandler_2` ?  is this a temporary name?

Reply:
Yes, I need to refactor and migrate a lot of functionality from the current Basehandler which contained everything. It will become the new Basehandler once that is done.